psi4/driver/qcdb/libmintsbasisset.py

```diff
@@ -808,7 +808,7 @@ def construct(cls, parser, mol, key, deffit=None, basstrings=None, return_atomli
                         tmp.append(univdef[deffit])
                     seek['basis'] = [item for item in tmp if item != (None, None, None)]
                     seek['entry'] = [symbol]
-                    seek['path'] = libraryPath
+                    seek['path'] = basisPath
```

Comment:
I think this right, but defiantly needs a @loriab thumbs up.

Reply:
When the issue came up, I couldn't decide for several days if I was comfortable with the switch. Now ok with it.